test/package/packageTest.nut.ts

```diff
@@ -25,6 +26,13 @@ import { deletePackage } from '../../src/package';
 import { PackageVersion } from '../../src/package';
 import { Package } from '../../src/package';
 import { PackagingSObjects } from '../../src/interfaces';
+import { VersionNumber } from '../../src/utils/versionNumber';
+import { AncestryJsonProducer, PackageAncestry } from '../../src/package/packageAncestry';
+import { AncestryTreeProducer } from '../../src/package/packageAncestry';
+import {
+  AncestryRepresentationProducer,
```

Comment:
NIT: should be able to import all of these from `../../src/package` or `/src/interfaces` and combine a few above